### DISCUSSION TITLE
any idea whether the database contains information like the SES-CD simple endoscopy score for crohn's disease?

### DISCUSSION THREAD

**Discussion by bhomass** [MIT-LCP/mimic-code#1456] [MIMIC-III]
I have been studying MIMIC III. I have a requirement to look up the SES-CD score. Does this information exist in one of the tables? I did a search in the notes text, and did not find any matches.

**Comment by alistairewj** [comment]
Never seen anything like it unfortunately, but you could look at the *d_items* table.
